lib/preference/util.js

```diff
@@ -68,10 +68,11 @@ export default class PreferencesUtil {
     const sv = document.querySelector('.settings-view');
     for (var btn of sv.querySelectorAll('.meta-controls .install-button')) {
       if (!this.isAlreadyLocalized(btn)) {
-        var version;
+        // var version;
+        var version = btn.textContent.substr(9);
         const buttonText = btn.textContent.startsWith('Update to')
-          ? this.defS['control-btns'].update +
-            (version = btn.textContent.substr(9))
+          ? this.defS['control-btns'].update + version
+        //    (version = btn.textContent.substr(9))
```

Comment:
Thanks, I should have refactored these old codes 😃

Reply:
The old code cause CI error, so I modified them.